cmd/print.go

```diff
@@ -228,25 +229,25 @@ func newPrintCommand() *cobra.Command {
 	// generated from Ingress resources.
 	var cmd = &cobra.Command{
 		Use:   "print",
-		Short: "Prints HTTPRoutes and Gateways generated from Ingress resources",
-		RunE:  pr.PrintGatewaysAndHTTPRoutes,
+		Short: "Prints Gateway API objects generated from ingress and provider-specific resources",
+		RunE:  pr.PrintGatewayAPIObjects,
 	}
 
 	cmd.Flags().StringVarP(&pr.outputFormat, "output", "o", "yaml",
-		fmt.Sprintf(`Output format. One of: (%s)`, strings.Join(allowedFormats, ", ")))
+		fmt.Sprintf(`Output format. One of: (%s).`, strings.Join(allowedFormats, ", ")))
 
-	cmd.Flags().StringVar(&pr.inputFile, "input_file", "",
-		`Path to the manifest file. When set, the tool will read ingresses from the file instead of reading from the cluster. Supported files are yaml and json`)
+	cmd.Flags().StringVar(&pr.inputFile, "input-file", "",
```

Comment:
It's about time this one is changed
Please update the README with this change as well, as it currently mentions we have a `--input_file` flag

Reply:
yep, good catch. Updated in [d886474](https://github.com/kubernetes-sigs/ingress2gateway/pull/156/commits/d886474abeaf455aec00f9b818f979b6b422f99f)